features/payment/managing_payment_methods/payment_method_unique_code_validation.feature

```diff
@@ -0,0 +1,18 @@
+@managing_payment_methods
+Feature: Payment method unique code validation
+    In order to avoid adding payment method with the same code
+    As an Administrator
+    I want to be prevented from adding a new payment method with an existing code
+
+    Background:
+        Given the store has a payment method "Offline" with a code "offline"
+        And I am logged in as an administrator
+
+    @todo
+    Scenario: Trying to add a new payment method with used code
+        Given I want to create a new payment method
+        When I name it "Cash on delivery"
+        And I specify its code as "offline"
```

Comment:
Code here should be something like ``cash_on_delivery``, the gateway should be ``offline``.